packages/@uppy/core/src/locale.ts

```diff
@@ -12,6 +12,8 @@ export default {
       0: 'You have to select at least %{smart_count} file',
       1: 'You have to select at least %{smart_count} files',
     },
+    aggregateExceedsSize:
```

Comment:
Shall we add it for too for the languages we know?

For dutch it's
> Je hebt %{size} in bestanden geselecteerd maar %{sizeAllowed} is maximaal toegestaan

Reply:
I'd like to do all translations in further PRs after #5050 lands, I think the phrasing/UI is still up to discussion.